pkg/virt-operator/install-strategy/create.go

```diff
@@ -1415,6 +1418,66 @@ func addOrRemoveSSC(targetStrategy *InstallStrategy,
 	return nil
 }
 
+func syncPodDisruptionBudgetForDeployment(deployment *appsv1.Deployment, clientset kubecli.KubevirtClient, kv *v1.KubeVirt, expectations *util.Expectations, stores util.Stores) error {
+	podDisruptionBudget := components.NewPodDisruptionBudgetForDeployment(deployment)
+
+	imageTag := kv.Status.TargetKubeVirtVersion
+	imageRegistry := kv.Status.TargetKubeVirtRegistry
+
+	injectOperatorMetadata(kv, &podDisruptionBudget.ObjectMeta, imageTag, imageRegistry)
```

Comment:
related to an earlier comment: here you would also potentially modify the deployment labels, if you don't use a copy. In this case not an issue, because it will just set an already existing label, but you never know...